workers/genotype_extractor.py

```diff
@@ -21,6 +21,11 @@
 
 @worker.task(bind=True)
 def extract(self, vcf_id):
+    register_running_task(self, vcf_id)
+    return _extract(vcf_id)
```

Comment:
I just think of it as "don't rely on this method; it's not part of the API, but internal to the module".